### DISCUSSION TITLE
Baby name generator

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1140] [Ideas]
Sometimes I find myself at a loss for a baby name. A name generator, which would spit out a bunch of names, would be very useful in such a moment.

It could have names from various species (human, dwarf), races (caucasian, aboriginal), sexes, and cultures (russian, british).

It could store the lists of potential names in simple text files; one entry per line. Or it could a huge sorted json (to allow for easy inclusiaon in multiple catagories) file for what's applicable to a name.

An alternative is to allow for importing lists of baby names.

**Comment by vkbo** [comment]
I use name generators all the time to get ideas for names for secondary characters, but I don't really see a value in adding them as a feature. There are tons of websites that do this, both for baby names, for fictional characters, and for fantasy. The scope here is huge, and there are so many narrow options people would like to make depending on what exactly sort of text they're working on.
